closed/src/java.base/share/classes/java/security/CRIUConfigurator.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2022, 2022 All Rights Reserved
+ * ===========================================================================
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * IBM designates this particular file as subject to the "Classpath" exception
+ * as provided by IBM in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+ *
+ * ===========================================================================
+ */
+
+package java.security;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import sun.security.action.GetPropertyAction;
+import sun.security.jca.ProviderList;
+import sun.security.jca.Providers;
+
+/**
+ * Configures the security providers when in CRIU mode.
+ */
+public final class CRIUConfigurator {
+    private static Properties systemProps;
+    /** Stores the old security providers (position, name). */
+    private static HashMap<String, String> oldProviders = new Hashtable<>();
```

Comment:
This won't compile (`Hashtable` is not assignable to `HashMap`); it should be:
```
    private static final Map<String, String> oldProviders = new HashMap<>();
```

Reply:
sorry, missed that
changed it